app/assets/javascripts/application.js

```diff
@@ -57,3 +57,5 @@ $(document).ready(function() {
         delegateSelector: 'a.smoothscroll'
     });
 });
+
+UnobtrusiveFlash.flashOptions['timeout'] = 2000;
```

Comment:
> That's 2 seconds, is that really enough time for someone to read a comment? 

It would vary person to person. Flash message have at max 3 words except for resource name. Apart from this, the reason I felt to keep it 2 second was just to assure the least number of flash message to appear on the page at any instant. More the number of messages, more resource content table would move towards bottom of page.